app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java

```diff
@@ -29,12 +31,16 @@
 import java.io.FileOutputStream;
 
 import java.io.IOException;
+import java.util.ArrayList;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
+import utils.FormatStringUtils;
+import utils.StringUtils;
+import utils.ViewUtils;
 
 
-public class TeleopActivity extends AppCompatActivity {
+public class TeleopActivity extends AppCompatActivity implements View.OnKeyListener {
 
     public String heading, auton, teleop, message;
```

Comment:
Only auton is being used remove the rest. 

Also put the String variables under the @BindViews